redisgraph/query_result.py

```diff
@@ -124,6 +126,12 @@ def parse_edge(self, cell):
         properties = self.parse_entity_properties(cell[4])
         return Edge(src_node_id, relation, dest_node_id, edge_id=edge_id, properties=properties)
 
+
+    def parse_path(self, cell):
+        nodes = self.parse_scalar(cell[0])
+        relationships = self.parse_scalar(cell[1])
+        return Path(nodes, relationships)
```

Comment:
Assert both nodes and relationships types, consider performing the assertion in Path's constructor